src/visitable.cpp

```diff
@@ -949,6 +949,22 @@ void location_visitable<monster>::remove_items_with( const
         mon->get_tied_item()->attempt_detach( check_item );
     }
 }
+/** @relates visitable */
+
+template <typename T>
+static int charges_of_ups( const T &self, int limit,
+                           const std::function<bool( const item & )> &filter,
+                           std::function<void( int )> visitor )
```

Comment:
```suggestion
                           const std::function<bool( const item & )> &,
                           std::function<void( int )> )
```
these parameters doesn't do anything